### ISSUE TITLE
Redesign dotty.epfl.ch/index.html

### ISSUE THREAD

**Issue by TheElectronWill** [lampepfl/dotty#7642]
After #7595, the docs got a new theme (implemented in #7612).
The front page still needs to be changed to something more attractive (and that fits the colors and fonts of the new docs).

I think it could contain (not necessarily in that order):
- the logo and label (of course)
- links to github, docs, etc.
- a quick overview of some new features. IIRC Martin Odersky did a poll on Twitter some time ago to elect the "top" dotty features, and also gave his own opinion. We could compare Scala 2  code with Dotty code to emphasize the improvement.
- a list of some (other) features
- how to install dotty
- how to start using it (sbt, scastie)

**Comment by AleksanderBG** [comment]
@TheElectronWill Thanks for your work! In the interest of keeping our issue queue clean, I'm going to close this issue. If you want to discuss a related topic, feel free to open another issue or just a PR!
